pwm/servo/main.c

```diff
@@ -0,0 +1,19 @@
+#include "pico/stdlib.h"
+#include "servo.h"
+#include <stdbool.h>
+
+int main()
+{
+    for(;;)
+    {
+        servo_init(1, 50);
```

Comment:
I've not tried running this code, but I _suspect_ that `servo_init` probably ought to be before the `for` loop, rather than inside it?

Reply:
oh yeah sorry forgot